fendermint/vm/interpreter/src/chain.rs

```diff
@@ -533,7 +646,15 @@ where
                     // Check that blobs that are being enqueued are still in "added" state in the actor
                     // Once we enqueue a blob, the actor will transition it to "pending" state.
                     if !is_blob_added(&mut state, blob.subscriber, blob.hash, blob.id)? {
-                        tracing::debug!(hash = %blob.hash, "blob is not added onchain; rejecting proposal");
+                        tracing::warn!(hash = %blob.hash, "blob is not added onchain; rejecting proposal");
+                        return Ok(false);
+                    }
+
+                    // Reject the proposal if the current processor is not keeping up with blob
+                    // resolving.
```

Comment:
give slow processors the opportunity to reject

Reply:
makes sense to do this here but does it mean that new block will be created more slowly if proposals are rejected?

Reply:
if the block doesn't have enough votes, yes, it would have to advance to another voting "round". if the rejector doesn't have much relative power, the block is still accepted.